WolvenKit.App/ViewModels/Dialogs/SoundModdingViewModel.cs

```diff
@@ -79,26 +79,28 @@ private void LoadInfo()
         }
 
         var path = Path.Combine(Environment.CurrentDirectory, "Resources", "soundEvents.json");
-        path = Path.Combine(_projectManager.ActiveProject.ResourcesDirectory, "info.json");
```

Comment:
Now it reads the wrong file. Should be info.json, not soundEvents.json

Reply:
Oh, alright! I was guessing from context, but you actually know. Fixing :)